doc/nrf/releases_and_maturity/abi_compatibility.rst

```diff
@@ -47,57 +47,41 @@ The following table illustrates ABI compatibility between different versions of
 ABI compatibility ensures that the Secure Domain and System Controller firmware binaries do not need to be recompiled each time the application, radio binaries, or both are recompiled, as long as they are based on a compatible |NCS| version.
 Additionally, maintaining ABI compatibility allows the nRF54H20 SoC binary components to work together without recompilation when updating to newer |NCS| versions.
 
-nRF54H20 SoC Binaries v0.8.0 changelog
+nRF54H20 SoC Binaries v0.9.0 changelog
 **************************************
 
 The following sections provide detailed lists of changes by component.
 
-Secure Domain Firmware (SDFW) v9.0.0
+Secure Domain Firmware (SDFW) v10.0.0
 =====================================
 
 Added
 -----
 
-* Purge protection can be enabled over ADAC.
-* Clock control is enabled in SDFW.
-* Global domain power request service is integrated in SDFW.
-* PUF values from SDROM are cleared on boot.
+* SUIT in-place updatable components (IPUC) are now available to local domains over IPC through the SSF IPUC service.
```

Comment:
This feature is apparently meant to be exposed through a higher level software component which is not ready yet, so it may not be worth mentioning here